release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3646,3 +3646,9 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.14.0 --pullspecs
 ----
 //replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+
+[id="ocp-4-14-0-bug-fixes"]
+==== Bug fix
```

Comment:
This would with the `Networking` bug [fixes](https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-bug-fixes:~:text=OCPBUGS%2D11889)-,Networking,-Previously%2C%20when%20a) that shipped with 4.14.0

Reply:
Sorry I am  confused @JoeAldinger I was asked to add it here as it was not available in 4.14 GA but was fixed in this z stream but did not get mentioned as the security level was set incorrectly.

Reply:
@kquinn1204 the bug text should go with the rest of the GA bug fixes instead of making a new bug fix section. The bug is attached to the 4.14.0 advisory, not a 4.14.z advisory.